examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -1125,13 +1125,13 @@ void pxScene2d::onUpdate(double t)
 
       // TODO:  update / render times need some work...
 
-      // double draw_ms   = ( (double) sigma_draw     / (double) frameCount ) * 1000.0f; // Average frame  time
-      // double update_ms = ( (double) sigma_update   / (double) frameCount ) * 1000.0f; // Average update time
+      double draw_ms   = ( (double) sigma_draw     / (double) frameCount ) * 1000.0f; // Average frame  time
+      double update_ms = ( (double) sigma_update   / (double) frameCount ) * 1000.0f; // Average update time
 
-      // rtLogDebug("%g fps   pxObjects: %d   Draw: %g   Tex: %g   Fbo: %g     draw_ms: %0.04g   update_ms: %0.04g\n",
-      //     fps, pxObjectCount, dpf, bpf, fpf, draw_ms, update_ms );
+      rtLogError("%d fps   pxObjects: %d   Draw: %g   Tex: %g   Fbo: %g     draw_ms: %0.04g   update_ms: %0.04g\n",
```

Comment:
Is this change to Error and being commented in due to local debugging, but shouldn't be in PR?

Reply:
The code is no longer commented - but compiled out by a #define 

It’s inactive - unless developer builds it

Reply:
Any additional changes/comments ?